Packs/CloudIncidentResponse/Playbooks/playbook-Cortex_XDR_-_Cloud_Cryptomining.yml

```diff
@@ -1163,6 +1163,51 @@ inputs:
   required: false
   description: Determines whether to convert the IP address to a hostname using a DNS query (True/ False).
   playbookInputQuery:
+inputSections:
+- inputs:
+  - incident_id
+  - alert_id
+  - cloudProvider
+  name: Incident Data
+  description: Incident details and metadata.
+- inputs:
+  - SOCEmailAddress
+  - requireAnalystReview
+  name: Incident Management
```

Comment:
I think we can combine the Incident Data and the Incident management sections together

Reply:
done